Sources/CXUtility/Locking.swift

```diff
@@ -1,21 +1,23 @@
+#if !COCOAPODS
 import CXLibc
+#endif
 
 public protocol Locking {
-    
```

Comment:
It seems that you accidentally formatted the entire file. Please keep changes minimal.